docs/tutorials/kubernetes-cluster-in-aws-with-gardener/_index.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Can you describe in the PR description from exactly where you are moving this tutorial? For example a reviewer may want to check whether everything is moved properly, etc.